src/lint/linter/WebCliLinter.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * Uses UberStandard to detect errors and potential problems in JavaScript code.
+ */
+final class WebCliLinter extends ArcanistExternalLinter {
```

Comment:
I'd suggest prefix class and file names with `Uber` (`UberWebCliLinter`). This will make easier to separate which classes were added by us.